common/test/acceptance/pages/lms/fields.py

```diff
@@ -211,6 +211,15 @@ def link_title_for_link_field(self, field_id):
         query = self.q(css='.u-field-link-title-{}'.format(field_id))
         return query.text[0] if query.present else None
 
+    def wait_for_link_title_for_link_field(self, field_id, expected_title):
+        """
+        Wait until the title of the specified link field equals expected_title.
+        """
+        return EmptyPromise(
+            lambda: self.link_title_for_link_field(field_id) == expected_title,
+            "Link field with link title \"{0}\" is visible.".format(expected_title)
```

Comment:
Just a note (totally don't have to fix this) but in python since you can define a string with single or double quotes, if you define it with single quotes you don't have to escape the double quotes (and vice versa).